### PR TITLE
Add spotify support

### LINKED ISSUE #131: Spotify Support
Add support for loading songs and playlists from Spotify using https://github.com/MenuDocs/erela.js-spotify

### PR DESCRIPTION
**Pull request by AjayACST**
This will close #131

### CODE REVIEW

**Comment by AjayACST** [comment]
@Darkflame72 Could you please review this?

File: commands/play.js [hunk 1]
```diff
@@ -47,6 +47,8 @@ module.exports = {
       );
     await interaction.editReply({ embeds: [searchEmbed] });
 
+    console.log(query);
```
Comment (Darkflame72) [inline commands/play.js:50]: Does this need to be logged?

File: deploy.js [hunk 2]
```diff
@@ -46,12 +46,13 @@ async function registerCommands() {
...
       name: "playlist",
-      description: "Plays all songs from a youtube playlist.",
+      description:
+        "Plays all songs from a youtube playlist or spotify album or playlist.",
       options: [
         {
           name: "playlisturl",
           type: "STRING",
-          description: "Youtube URL to playlist",
+          description: "Link to youtube playlist or spotify album or playlist.",
```
Comment (Darkflame72) [inline deploy.js:55]: Is there a difference between a Spotify album or playlist?

File: index.js [hunk 3]
```diff
@@ -32,6 +35,12 @@ Sentry.init({
   environment: sentryEnv,
 });
 
+// let spotifyPlugin
+
+// if (spotifyEnabled) {
+//   spotifyPlugin =
+// }
+
```
Comment (Darkflame72) [inline index.js:43]: Should this be removed?

**Review by Darkflame72 — CHANGES_REQUESTED** [review]
I have left some comments with some questions but it looks good and functions well

File: deploy.js (hunk 2 above)
Comment (AjayACST) [reply deploy.js:55]: No but it's just to clarify that you can do either with that command.

**Review by Darkflame72 — APPROVED** [review]

**Pull request merged**